File: MATEMÁTICA/2516/main.py
from sys import stdin


def elapsed(s, va, vb):
    return s / (va - vb)


def formatted_print(time):
    print('impossivel' if time < 0 else f'{time:.2f}')


for line in stdin:
    s, va, vb = map(int, line.split())
    formatted_print(elapsed(s, va, vb))

File: MATEMÁTICA/2516/main.js
function elapsed(s, va, vb) {
    return s / (va - vb);
}

function formattedPrint(time) {
    console.log(time < 0 ? 'impossivel' : time.toFixed(2));
}

let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').trim().split('\n');
for (let line of lines) {
    let [s, va, vb] = line.split(' ').map(Number);
    formattedPrint(elapsed(s, va, vb));
}
